GiniVision/Classes/CameraViewController.swift

```diff
@@ -95,7 +100,12 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
         guard let image = cameraOverlayImage, let cgImage = image.cgImage else {
             return nil
         }
-        return UIImage(cgImage: cgImage , scale: 1.0, orientation: UIDevice.current.orientation.isLandscape ? .right : UIImageOrientation.up)
+        if UIDevice.current.orientation.isFlat {
+            return UIImage(cgImage: cgImage , scale: 1.0, orientation: UIApplication.shared.statusBarOrientation.isLandscape ? .right : UIImageOrientation.up)
```

Comment:
Why does one return statement use the status bar orientation and the other, the device orientation?